python/feathub/processors/spark/ast_evaluator/spark_ast_evaluator.py

```diff
@@ -56,9 +56,10 @@ def eval_value_node(self, ast: ValueNode, variables: Optional[Dict]) -> Any:
         return str(ast.value)
 
     def eval_func_call_op(self, ast: FuncCallOp, variables: Optional[Dict]) -> Any:
-        # TODO: Add support for Feathub built-in functions.
         args = [self.eval(v, variables) for v in ast.args.values]
-        return f"{ast.func_name}({', '.join(args)})"
+        if ast.func_name.upper() == "UNIX_TIMESTAMP":
+            return f"TO_UNIX_TIMESTAMP({', '.join(args)})"
```

Comment:
Both Spark/Flink and the FeatHub implementation of this build-in function accept the timestamp format as the 2nd parameter. Could you help update our wiki page to specify this? Maybe use Spark' doc as example?

[1] https://github.com/alibaba/feathub/blob/master/docs/feathub_expression.md
[2] https://spark.apache.org/docs/latest/api/sql/index.html#to_unix_timestamp